src/main/java/net/mcreator/ui/dialogs/preferences/PreferencesDialog.java

```diff
@@ -50,6 +50,7 @@
 
 	DefaultListModel<String> model = new DefaultListModel<>();
 	JPanel preferences = new JPanel();
+	private final ThemesPanel themes; // We create the variable here, so we can access it when we store tha preferences

```

Comment:
Remove this comment as it does not add much value to the understanding of the code and could even be confusing to some